aci/data_source_aci_cloudlb.go

```diff
@@ -150,6 +150,12 @@ func dataSourceAciCloudL4L7LoadBalancer() *schema.Resource {
 				Computed: true,
 				Set:      schema.HashString,
 			},
+			"static_ip_address": &schema.Schema{
```

Comment:
Should this attribute be added to the documentation of aci_cloud_l4_l7_native_load_balancer? When done in legacy docs, make sure to run go generate to have it correctly copied.

Reply:
added